client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -430,3 +434,23 @@ export const useCommandsArrayWpcom = ( {
 
 	return commands;
 };
+
+export const commandNameDisplayLabel: CommandNameDisplayLabel = {
+	openSiteDashboard: 'Open site dashboard',
+	openHostingConfiguration: 'Open hosting configuration',
+	openPHPmyAdmin: 'Open database in phpMyAdmin',
+	manageDns: 'Manage DNS records',
+	copySshConnectionString: 'Copy SSH connection string',
+	openSshCredentials: 'Open SFTP/SSH credentials',
+	resetSshSftpPassword: 'Reset SSH/SFTP password',
+	openSiteStats: 'Open site stats',
+	openActivityLog: 'Open activity log',
+	openBackups: 'Open backups',
+	viewSiteMetrics: 'View site metrics',
+	openPHPLogs: 'Open PHP logs',
+	openWebServerLogs: 'Open web server logs',
+	manageStagingSites: 'Manage staging sites',
+	managePHPVersion: 'Manage PHP version',
+	manageCacheSettings: 'Manage cache settings',
+	manageAdminInterfaceStyle: 'Manage admin interface style',
```

Comment:
These strings should be translated.

Reply:
I have removed them in the refactored version and added them with translation to the command definitions